File: Java/src/modelo/estudiante.java
public class estudiante {
    private String id;
    private String nombre;
    private double promedio;

    public estudiante(String id, String nombre, double promedio) {
        this.id = id;
        this.nombre = nombre;
        this.promedio = promedio;
    }

    public String getId() { return id; }
    public String getNombre() { return nombre; }
    public double getPromedio() { return promedio; }

    public void setNombre(String nombre) { this.nombre = nombre; }
    public void setPromedio(double promedio) { this.promedio = promedio; }

    @Override
    public String toString() {
        return "ID: " + id + " | Nombre: " + nombre + " | Promedio: " + promedio;
    }
}

File: CPP/modelo/estudiante.cpp
#include <iostream>
#include <string>

class estudiante {
private:
    std::string id;
    std::string nombre;
    double promedio;

public:
    estudiante() : id(""), nombre(""), promedio(0.0) {}
    estudiante(std::string id, std::string nombre, double promedio) 
        : id(id), nombre(nombre), promedio(promedio) {}

    std::string getId() const { return id; }
    std::string getNombre() const { return nombre; }
    double getPromedio() const { return promedio; }

    void setNombre(std::string n) { nombre = n; }
    void setPromedio(double p) { promedio = p; }

    void mostrarInfo() const {
        std::cout << "ID: " << id << " | Nombre: " << nombre << " | Promedio: " << promedio << std::endl;
    }
};